docs-generation/scripts/sync_frontend_translations.py

```
@@ -108,34 +179,51 @@ def create_translation_report(frontend_path):
                         if param_trans['tooltip']:
                             print(f"      Tooltip: {param_trans['tooltip'][:60]}...")
 
-def export_translation_json(frontend_path, output_file=None):
+def export_translation_json(frontend_path, output_file=None, use_remote=False):
```

Ruff (0.16.1)

[warning] Boolean default positional argument in function definition

(FBT002)